dnf-docker-test/features/steps/repo_steps.py

```diff
@@ -83,6 +83,18 @@ def parse_enable_disable(text):
 
 register_type(enable_disable=parse_enable_disable)
 
+@parse.with_pattern(r"local |http |ftp |")
```

Comment:
That's what I started with but I didn't want to loose the previous/default scenario whe the type is not specified at all.